extensions/wikia/LocalSitemapPage/LocalSitemapPage.setup.php

```diff
@@ -12,4 +12,3 @@
 // hooks
 $wgHooks['ArticleFromTitle'][] = 'LocalSitemapPageHooks::onArticleFromTitle';
 $wgHooks['PageHeaderActionButtonShouldDisplay'][] = 'LocalSitemapPageHooks::onPageHeaderActionButtonShouldDisplay';
-$wgHooks['PageHeaderLanguageSelectorShouldDisplay'][] = 'LocalSitemapPageHooks::onPageHeaderLanguageSelectorShouldDisplay';
```

Comment:
Currently it's not necessary as we only show language selector for pages where `count( $this->languageList ) > 0` is `true`. On local sitemap it's true.